pytorch_lightning/utilities/cloud_io.py

```diff
@@ -25,9 +25,13 @@
 
 
 def load(path_or_url: str, map_location=None):
-    if urlparse(path_or_url).scheme == "" or Path(path_or_url).drive:  # no scheme or with a drive letter
-        return torch.load(path_or_url, map_location=map_location)
-    return torch.hub.load_state_dict_from_url(path_or_url, map_location=map_location)
+    try:
+        fs = get_filesystem(path_or_url)
+    except ValueError:
+        # Raised when the filesystem is neither local nor registered.
+        return torch.hub.load_state_dict_from_url(path_or_url, map_location=map_location)
```

Comment:
@ananthsub noob question, can fsspec savely replace torch.hub?

Reply:
I believe the tests are failing because now the torch.hub.load is ignored when fsspec handles http.

Reply:
@awaelchli  this is super hacky but idk how to get fsspec and the unit tests to work with torch.hub.load. i think fsspec should be a direct replacement